README.md

```diff
@@ -110,6 +110,10 @@ The following must be recursively owned by the lorisadmin user and by Apache gro
    /data/incoming/
    /data/$projectname/bin/mri/dicom-archive/.loris_mri/prod
    ```
+8. Verify in the front-end some 'Imaging Pipeline' Configuration values
+
+As a final verification step, ensure that Config table entries accessible via the Configuration module in the front-end (Admin menu) under the `Imaging Pipeline` section are properly populated for ConfigSetting `Loris-MRI Data Directory` , `Study Name`, `User to notify when executing the pipeline`, `Full path to get_dicom_info.pl script`, and `Path to Tarchives`.
```

Comment:
This is great.  I'd further shorten the wording here, and list the specific settings with bullets. 
break these down into bullet points, to make the text more readable.